variables.tf

```diff
@@ -80,6 +80,11 @@ variable "create_policy" {
   description = "Set to true to create a new access group (using the value of var.access_group_name) with a VPN Client role"
   type        = bool
   default     = true
+
+  validation {
+    condition     = !(var.create_policy == true && var.access_group_name == null)
```

Comment:
Is this not easier to read?
`var.create_policy ? var.access_group_name != null : true`